### PR TITLE
MGMT-4181 Generate service image URL for non S3 enabled deployments

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -962,14 +962,21 @@ func (b *bareMetalInventory) updateImageInfoPostUpload(ctx context.Context, clus
 	cluster.ImageInfo.SizeBytes = &imgSize
 
 	// Presigned URL only works with AWS S3 because Scality is not exposed
+	signedURL := ""
```
Comment (avishayt) [inline internal/bminventory/inventory.go:965]:
```suggestion
	downloadURL := ""
```

Comment (nmagnezi) [reply internal/bminventory/inventory.go]: done

**Review by avishayt — APPROVED** [review]

**Pull request merged**